### ISSUE TITLE
Cannot find the config.js file

### ISSUE THREAD

**Issue by crucialfelix** [neumino/chateau#47]
'0.3.12'

fresh install
any command to chateau fails

```
--- Sites/crucialjs ‹master. M?› » chateau
```

```
# even giving it a path to the config fails
--- Sites/crucialjs ‹master. M?› » chateau -f config.js

module.js:340
    throw err;
          ^
Error: Cannot find module '../config.js'
    at Function.Module._resolveFilename (module.js:338:15)
    at Function.Module._load (module.js:280:25)
    at Module.require (module.js:364:17)
    at require (module.js:380:17)
    at module.exports (/usr/local/lib/node_modules/chateau/routes/api.js:7:18)
    at module.exports (/usr/local/lib/node_modules/chateau/app.js:7:38)
    at Object.<anonymous> (/usr/local/lib/node_modules/chateau/bin/chateau:18:31)
    at Module._compile (module.js:456:26)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
```

the config does exist.

I even tried moving it up a directory since that is where it is looking:

```
--- Sites/crucialjs ‹master. M?› » mv config.js ../config.js
--- Sites/crucialjs ‹master. M?› » chateau --version

module.js:340
    throw err;
          ^
Error: Cannot find module '../config.js'
    at Function.Module._resolveFilename (module.js:338:15)
    at Function.Module._load (module.js:280:25)
    at Module.require (module.js:364:17)
    at require (module.js:380:17)
    at module.exports (/usr/local/lib/node_modules/chateau/routes/api.js:7:18)
    at module.exports (/usr/local/lib/node_modules/chateau/app.js:7:38)
    at Object.<anonymous> (/usr/local/lib/node_modules/chateau/bin/chateau:18:31)
    at Module._compile (module.js:456:26)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
```

**Comment by grimen** [comment]
This is a bug, ../app.js is not working in node land magically, it should have been expanding the path to absolute path in the root file before passing it further - it only works to use chateau if you clone down the repo and rename the config file. When I am less burdened with work I will probably send ind a pull request on this.

**Comment by neumino** [comment]
I just did a clean install and in `/usr/lib/node_modules/chateau`, I just copied `config.template.js` to `config.js` and it worked fine.

```
michel@xone:~$ which chateau
/usr/bin/chateau
```
```
michel@xone:/usr/bin$ ls -al | grep chateau
lrwxrwxrwx 1 root root       39 Feb  7 18:27 chateau -> ../lib/node_modules/chateau/bin/chateau
```

**Comment by grimen** [comment]
@neumino It is a pretty dirty way of doing configuration as the files will be overwritten on update. I would suggest following POSIX conventions an auto-symlink to ~/.chateau.js or similar. Renaming things in `/usr/lib` is definitely not pretty.

The bug I refer to is right now doing `chateau -f ./hello.js`,  `chateau -f hello.js`,  `chateau -f /some/path/hello.js` all fails. Meaning the README is either wrong or it is a bug. I would say a bug as renaming system files or things in node_modules is not a good convention, or a convention seen at all.

**Comment by neumino** [comment]
Oh sorry I didn't see that `chateau -f ./hello.js` doesn't work.
I'll look into it this week, I should have some time.

**Comment by grimen** [comment]
@neumino That would be awesome.

**Comment by neumino** [comment]
Fixed in 0.3.13.

**Comment by neumino** [comment]
(thanks to @flienteen)
